File: christmas/december task 2/array tasks other languages/sum_integer.py
def sum_of(array, suming):

    for count in range (0, len(array)):

        suming += array[count]
    

    return suming;
   

array = [5, 4, 3, 6, 7];

suming = 0;

print(sum_of(array, suming));

File: christmas/december task 2/array tasks other languages/SumInteger.js
function sumOf(array, sum) {

    for(count = 0; count < array.length; count ++) {

        sum += array[count];
     }

    return sum;
    }

array = [5, 4, 3, 6, 7];

sum = 0;

console.log(sumOf(array, sum));
